react/features/virtual-background/components/VirtualBackgroundDialog.js

```diff
@@ -106,6 +108,151 @@ function _mapStateToProps(state): Object {
 
 const VirtualBackgroundDialog = translate(connect(_mapStateToProps)(VirtualBackground));
 
+const useStyles = makeStyles(theme => {
+    return {
+        dialog: {
+            marginLeft: '-10px',
+            position: 'relative',
+            maxHeight: '300px',
+            color: 'white',
+            display: 'inline-grid',
+            gridTemplateColumns: 'auto auto auto auto auto',
+            columnGap: '9px',
+            cursor: 'pointer',
+            [[ '& .desktop-share:hover',
+                '& .thumbnail:hover',
+                '& .blur:hover',
+                '& .slight-blur:hover',
+                '& .virtual-background-none:hover' ]]: {
+                opacity: 0.5,
+                border: '2px solid #99bbf3'
+            },
+            '& .background-option': {
+                marginTop: theme.spacing(2),
+                borderRadius: theme.shape.borderRadius,
+                height: '60px',
+                width: '107px',
+                textAlign: 'center',
+                justifyContent: 'center',
+                fontWeight: 'bold',
+                boxSizing: 'border-box',
+                display: 'flex',
+                alignItems: 'center'
+            },
+            '& thumbnail-container': {
+                position: 'relative',
+                '&:focus-within .thumbnail ~ .delete-image-icon': {
+                    display: 'block'
+                }
+            },
+            '& .thumbnail': {
+                objectFit: 'cover'
+            },
+            '& .thumbnail:hover ~ .delete-image-icon': {
+                display: 'block'
+            },
+            '& .thumbnail-selected': {
+                objectFit: 'cover',
+                border: '2px solid #246fe5'
+            },
+            '& .blur': {
+                boxShadow: 'inset 0 0 12px #000000',
+                background: '#7e8287',
+                padding: '0 10px'
```

Comment:
Please reuse styles. Create a class that contains all the common styles for blur classes and use that